.mvn/extensions.xml

```diff
@@ -0,0 +1,8 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
+  <extension>
+    <groupId>io.quarkus.bot</groupId>
+    <artifactId>build-reporting-maven-extension</artifactId>
```

Comment:
Any special reason to register the extension via this file and not via `pom.xml`?

Reply:
I didn't want to clutter the pom with this but... you're right, it's better this way and it avoids triggering the extension for the invoker calls so tests should pass. I pushed a change going in this direction, let's see how it goes.